### ISSUE TITLE
GET Course from "id" example doesn't work

### ISSUE THREAD

**Issue by paulxuca** [cobalt-uoft/documentation#32]
Going to https://cobalt.qas.im/api/1.0/courses/CSC148H1F20159?key="API_KEY_HERE" resolves to:
Error 400, message: A course with the specified identifier does not exist. Maybe another example would be better here.

**Comment by kashav** [comment]
Good catch – it's happening cause the API's using course data from 2016-17 now, which means changing the year should be enough to fix it.

We're going to end up having to do this for a lot of the APIs though, since many are date/session based (exams, athletics, shuttles, etc). I personally think it'd be better just to leave it as is and add a disclaimer saying the examples _might_ be out of date. Thoughts?

**Comment by paulxuca** [comment]
That's a good idea. I was thinking (in future implementations) it could be possible to pass the year/session as a parameter.

**Comment by qasim** [comment]
@paulxuca This is something we have the data to do but it hasn't been implemented. I should make some issues about it.

**Comment by paulxuca** [comment]
I could tackle that! Is all the data on the api server?

**Comment by qasim** [comment]
@paulxuca here's the corresponding issue: https://github.com/cobalt-uoft/cobalt/issues/41

The data is all in the datasets repository, with each session under a tag. You can see all the tags so far here: https://github.com/cobalt-uoft/datasets/tags

Let's discuss on the other issue what you need to accomplish this.
